main.tf

```diff
@@ -172,7 +184,7 @@ resource "aws_s3_bucket_server_side_encryption_configuration" "firehose-errors"
 resource "aws_cloudwatch_log_group" "kinesis" {
   #  checkov:skip=CKV_AWS_338:Short life error logs don't need long term retention
   #  checkov:skip=CKV_AWS_158:Default log encryption OK for short life error logs
-  name              = "/aws/kinesisfirehose/cloudwatch-to-s3-${random_id.name.hex}"
+  name              = "/aws/kinesisfirehose/cloudwatch-to-s3-${local.name_unique}"
```

Comment:
As with some prior resources, if a unique name is desirable then `name_prefix` would be a better choice for consistency

Reply:
I agree - but name is used in current version, so changing to name_prefix would break backward compatiblity